rero/rero-ils-ui
# menu: refactoring switch library

The switch library was only displayed on the root page of the
professional interface. This commit change this behavior and now this
menu is displayed on all pages of user has required right.
This commit also fix the dynamic population problem of the switch
library menu : When a new library was added, this library was not
dynamically added to the switch library menu. This commit fixes this
problem.

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
Co-Authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## Why are you opening this PR?


- Closes rero/rero-ils#821
- Closes rero/rero-ils#822
- Closes rero/rero-ils#1039
## How to test?

- Check issues to know how to test this commit
- Check the behavior of switch library menu in professional interface

## Code review check list
